### DISCUSSION TITLE
There is a way to disable prefetch on hover for static pages? This is overloading my server

### DISCUSSION THREAD

**Discussion by wilsonneto-swe** [vercel/next.js#26694] [Help]
Hi guys,
There is a way to disable prefetch on hover for static pages? This is overloading my server

Thanks so much!

**Comment by stubbies** [comment]
There is no way at the moment if you're using `<Link />`

https://github.com/vercel/next.js/blob/5b9ad8da909a7870280c39340c189392d6307994/packages/next/client/link.tsx#L299
